apps/cli/src/command/index.ts

```diff
@@ -6,11 +6,20 @@ import { ConvertCommand } from './convert.command';
 import { DevCommand } from './dev.command';
 import { DiffCommand } from './diff.command';
 import { DumpCommand } from './dump.command';
+import { IngressSyncCommand } from './ingress-sync.command';
 import { LintCommand } from './lint.command';
 import { PingCommand } from './ping.command';
 import { SyncCommand } from './sync.command';
 import { configurePluralize } from './utils';
 
+const versionCode = '0.17.0';
```

Comment:
How about setting this value in a configuration file or a separate constant file? This will help avoid forgetting to update it.

Reply:
This will be moved to a separate file later or injected at build time by the webpack DefinePlugin.